### PR TITLE
Images and inputs

### PR DESCRIPTION
**Pull request by rahulyadav139**
input and image components are added.

### CODE REVIEW

**Comment by SinchanaMS** [comment]
Looks good! I liked how the class names are self-explanatory.

File: palasio-design.css [hunk 1]
```diff
@@ -6,3 +6,5 @@
 @import url('./components/button/button.css');
 @import url('./components/badge/badge.css');
 @import url('./components/card/card.css');
+@import url('./components/image/image.css');
```
Comment (tarunsankhla) [inline palasio-design.css:9]: Looks good

**Review by tarunsankhla — APPROVED** [review]

**Comment by dev-ashishsethi** [comment]
in images section code snippet that you have put on webpage is of buttons and description is also of buttons

**Comment by dev-ashishsethi** [comment]
in input section code snippet that you have put on webpage is of buttons and description is also of buttons

**Comment by rahulyadav139** [comment]
> in input section code snippet that you have put on webpage is of buttons and description is also of buttons

Documentation is not perfect yet. I need to work on that.

**Pull request merged**